custom_components/sector/binary_sensor.py

```diff
@@ -27,6 +28,41 @@ async def async_setup_entry(
     devices = coordinator.data.get("devices", {})
     entities = []
 
+    # Define descriptions for each type of binary sensor using the generic description
+    SENSOR_DESCRIPTIONS = (
+        GenericSensorEntityDescription(
+            key="closed",
+            sensor_type="closed",
```

Comment:
`sensor_type` is same as `key` so we can remove it and use `key` only

Reply:
All sensor_types removed